### PR TITLE
Class Questions completed

### PR DESCRIPTION
**Pull request by ChaitanyaKDec19**

### CODE REVIEW

File: src/chaitanyaKDec19/ClassQuestions/OddEvenSumTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package chaitanyaKDec19.ClassQuestions;
+
+public class OddEvenSumTest {
+
+	public static void main(String[] args) {
+
+		if (args.length == 0) {
+			System.out.println("Please enter valid arguments i.e. numbers");
```
Comment (harshvegada) [inline src/chaitanyaKDec19/ClassQuestions/OddEvenSumTest.java]: Good point @ChaitanyaKDec19 
now you are thinking on various perceptive. keep it up

File: src/chaitanyaKDec19/ClassQuestions/PrimeNumSumTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+		int i;
+
+		for (i = 2; i <= num / 2; i++) {
+
+			if (num % i == 0) {
+				break;
+			}
+		}
+		if (i == (num / 2 + 1)) {
```
Comment (harshvegada) [inline src/chaitanyaKDec19/ClassQuestions/PrimeNumSumTest.java]: you can use flag here as well, so other can understand your logic.

File: src/chaitanyaKDec19/ClassQuestions/OddEvenSumTest.java (hunk 1 above)
Comment (ChaitanyaKDec19) [reply src/chaitanyaKDec19/ClassQuestions/OddEvenSumTest.java]: Thanks a lot Harsh :) @harshvegada

File: src/chaitanyaKDec19/ClassQuestions/PrimeNumSumTest.java (hunk 2 above)
Comment (ChaitanyaKDec19) [reply src/chaitanyaKDec19/ClassQuestions/PrimeNumSumTest.java]: Sure, next time I will keep it in mid

**Pull request merged**